templates/docs/examples/patterns/notifications/variants.html

```diff
@@ -0,0 +1,269 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Notifications / Variants {% endblock %}
+
+{% block standalone_css %}patterns_notifications{% endblock %}
+
+{% block style %}
+<link rel="stylesheet" href="{{ versioned_static('build/css/standalone/patterns_icons-additional.css') }}" />
+{% endblock %}
+
+{% block content %}
+<h4>Meta - Action buttons</h4>
```

Comment:
Would be good to have examples that are not implemented marked in some way, so it's easier to know which should work and which don't.